Purchases/FoundationExtensions/JSONDecoder+Extensions.swift

```diff
@@ -83,7 +83,6 @@ extension KeyedDecodingContainer {
         do {
             return try decode(type, forKey: key)
         } catch {
-            ErrorUtils.logDecodingError(error)
```

Comment:
Is this used anymore?

Reply:
Yes it's used in JSONDecoder.decode. But that one throws an error instead of using a default value so I think it makes sense to keep it there